content/200-orm/800-more/450-migrating-to-prisma/01-migrate-from-typeorm.mdx

```diff
@@ -862,7 +862,7 @@ export async function createDraftAction(req, res) {
 }
 ```
 
-Note that Prisma Client's nested write here save an initial query where first the `User` record needs to be retrieved by its `email`. That's because, with Prisma you can connect records in relations using any unique property.
+Note that Prisma Client's nested write here save an initial query where first the `User` record needs to be retrieved by its `email`. That's because, with Prisma ORM you can connect records in relations using any unique property.
```

Comment:
```suggestion
Note that Prisma Client's nested write here save an initial query where first the `User` record needs to be retrieved by its `email`. That's because, with Prisma Client you can connect records in relations using any unique property.
```